ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/BlockHeader.java

```diff
@@ -142,33 +144,50 @@ public void writeTo(final RLPOutput out) {
     out.writeBytes(extraData);
     out.writeBytes(mixHash);
     out.writeLong(nonce);
-
+    if (baseFee != null) {
```

Comment:
Should this be guarded with the feature flag?  Either on the parameter or the constructor.

Reply:
Sure. I will do that when step 0 gets merged.

Reply:
Done in https://github.com/hyperledger/besu/pull/619/commits/7d7f59b2d35fcf6fb4ec6853a732de388a52859e